packages/react/src/ButtonGroup/ButtonGroup.features.stories.tsx

```diff
@@ -73,7 +73,7 @@ export const InactiveButtonsGroup = () => {
   )
 
   const secondaryButton = (
-    <IconButton aria-label="Secondary Button Aria Label" aria-disabled={true} inactive={true} icon={DashIcon} />
+    <IconButton aria-label="Secondary Button" aria-disabled={true} inactive={true} icon={DashIcon} />
```

Comment:
Cleans up bad `aria-label` and ensures that either `aria-label` or `aria-labelledby` is provided on the IconButton (not both).